.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 /community-connectors/meetup/BUILD
+yarn-error.log
```

Comment:
Nit: recommend using https://www.gitignore.io/api/node as a base, and adding dscc-specific stuff